Lib/ruby/boost_shared_ptr.i

```diff
@@ -24,7 +24,9 @@
 
 // Typemap customisations...
 
+//
```

Comment:
I'd like the boost_shared_ptr.i typemaps to continue to be easily diff'd, so please remove the whitespace and comment changes unless they are replicated in all of the boost_shared_ptr.i files.